tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -563,6 +567,43 @@ def test_12_error_handling_invalid_roles(self, anthropic_client, test_config):
         assert_valid_error_response(error, "tester")
         assert_error_propagation(error, "anthropic")
 
+    @skip_if_no_api_key("anthropic")
+    def test_13_streaming(self, anthropic_client, test_config):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_13_streaming`

Add return type annotation: `None`

(ANN201)